ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/ExceptionManagement.selectors.ts

```diff
@@ -0,0 +1,5 @@
+export const selectors = {
+    tableSortColumn: (columnName: string) => `table th.pf-c-table__sort:contains("${columnName}")`,
+    tableColumnSortButton: (columnName: string) =>
+        `table th.pf-c-table__sort button:contains("${columnName}")`,
+};
```

Comment:
Just an idea for these selector files. If we export the entire object `as const`, and declare the strings returned by each simple function `as const`, we'll get really precise tooltips on hover in VSCode.
```suggestion
export const selectors = {
    tableSortColumn: (columnName: string) =>
        `table th.pf-c-table__sort:contains("${columnName}")` as const,
    tableColumnSortButton: (columnName: string) =>
        `table th.pf-c-table__sort button:contains("${columnName}")` as const,
} as const;
```

![image](https://github.com/stackrox/stackrox/assets/1292638/c7c78a27-243b-4c38-9ace-c7e778b66564)

![image](https://github.com/stackrox/stackrox/assets/1292638/e5953cbb-0051-406e-b0a0-ee27d98df825)

Reply:
Looking at this more, there are benefits in having the tooltip explicitly show the template literals, but there are some downsides here. Adding the `as const` after each selector can be cumbersome and reduce readability. I do like the idea though, so I was thinking that we could leave it as is and reevaluate making that change if we find it improves things. Interested to see what others think (cc: @dvail @vjwilson @alwayshooin @pedrottimark @bradr5 )

Reply:
I haven't tested this idea, but perhaps we only need the `as const` on the whole object, and not also on each individual property line?

```suggestion
export const selectors = {
    tableSortColumn: (columnName: string) => `table th.pf-c-table__sort:contains("${columnName}")`,
    tableColumnSortButton: (columnName: string) =>
        `table th.pf-c-table__sort button:contains("${columnName}")`,
} as const;
```

Reply:
The `as const` would be necessary on each property to get the hint. Otherwise, we'll just get the string.

Reply:
The `as const` would be necessary on each property to get the hint. Otherwise, we'll just get a string (first image).

Reply:
I found that adding `as const` to the object as a whole will give us the selector in the tooltip _only_ if the object property is a plan string. We would need the `as const` on each individual property that was a function. (Technically we need the `as const` on the string returned by the function.)